### PR TITLE
Modified a bit of CSS of gallery.html and index.html

### PR DESCRIPTION
**Pull request by Shreyan-12345**

### CODE REVIEW

File: gallery.html [hunk 1]
```diff
@@ -42,6 +42,14 @@ <h2 id="gallery_heading" class="text-center">Gallery</h2>
         </div>
 
     </section>
+    <style>
```
Comment (niloysikdar) [inline gallery.html:45]: Why are you adding style here if you are adding the same thing in the gallery.css file?

**Review by niloysikdar — COMMENTED** [review]
@Shreyan-12345

**Review by niloysikdar — CHANGES_REQUESTED** [review]
@Shreyan-12345

File: gallery.html (hunk 1 above)
Comment (Shreyan-12345) [reply gallery.html:45]: by mistake I did this , same thing is done in CSS file

**Pull request closed without merging**